public/app/percona/dbaas/components/DBCluster/DBClusterParameters/DBClusterParameters.test.tsx

```diff
@@ -16,13 +16,16 @@ describe('DBClusterParameters::', () => {
     const memory = root.find('[data-qa="cluster-parameters-memory"]');
     const cpu = root.find('[data-qa="cluster-parameters-cpu"]');
     const disk = root.find('[data-qa="cluster-parameters-disk"]');
+    const expose = root.find('[data-qa="cluster-parameters-expose"]');
```

Comment:
Can we use the `dataQa` helper here from `@percona/platform-core`?

Reply:
Done!